### DISCUSSION TITLE
doris be进程运行过程中频繁发现系统CPU load高检查对应原因为数据存储文件系统hung住

### DISCUSSION THREAD

**Discussion by lklkxcxc** [apache/doris#12548] [A - General / Q&A]
## 现象：
doris be进程运行一段时间会出现系统cpu load高，检查发现be数据存储文件系统卡主，IO不正常。iostat查看对应be数据存储文件系统读写为0，但是util一直是100%，执行ls，df等命令查看数据存储文件系统会卡主，fuser -m 该文件系统只有be进程在访问。kill -9 be
进程后变成僵尸进程

## doris版本：1.1 trunk-f286bde

strace be进程结果：
![WechatIMG3162](https://user-images.githubusercontent.com/10216780/189835239-e059ef2c-ba2a-42f0-926b-b0b27b9c57a7.png)

**Comment by JNSimba** [comment]
It is recommended to upgrade to 1.1.2

**Comment by dataalive** [comment]
is there any disk io error in `dmesg -T` ? 
and you could list `iostat -x 1` result for more info
